### ISSUE TITLE
ptrace: Ptrace{Requests, Events, Options} are not in an enum

### ISSUE THREAD

**Issue by chaosagent** [nix-rust/nix#439]
https://github.com/nix-rust/nix/blob/master/src/sys/ptrace.rs#L14-L63

From CONVENTIONS.md:

```
We represent sets of constants that are intended as mutually exclusive arguments to parameters of functions by enumerations.
```

**Comment by kamalmarhubi** [comment]
Thanks for the report! Saw your #438 as well. Going to comment there.

**Comment by Susurrus** [comment]
I believe this is now resolved with #749.
